examples/Basic.svelte

```diff
@@ -1,7 +1,9 @@
 <script>
 import { Editor, h } from 'typewriter-editor';
+import { options } from 'typewriter-editor/lib/rendering/vdom';
 import Root from 'typewriter-editor/lib/Root.svelte';
 
+options.renderKeys = true;
```

Comment:
This uses a debugging option to view `data-key="xyz"` in the HTML to visually see how the IDs are being used.